package.json

```diff
@@ -31,19 +31,26 @@
     "postversion": "git push && git push --tags"
   },
   "devDependencies": {
-    "@testing-library/cypress": "8.0.1",
+    "@testing-library/cypress": "8.0.2",
     "@types/cypress-image-snapshot": "3.1.6",
-    "@types/jest": "27.0.2",
-    "@types/node": "16.10.2",
-    "cypress": "8.5.0",
+    "@types/jest": "27.0.3",
+    "@types/node": "16.11.12",
+    "cypress": "9.1.1",
     "cypress-image-snapshot": "4.0.1",
-    "eslint": "7.32.0",
-    "eslint-config-galex": "2.16.12",
+    "eslint": "8.4.1",
+    "eslint-config-galex": "3.3.5",
     "identity-obj-proxy": "3.0.0",
-    "jest": "27.2.4",
+    "jest": "27.4.3",
     "jest-transform-stub": "2.0.0",
-    "prettier": "2.4.1",
-    "ts-jest": "27.0.5",
-    "typescript": "4.4.3"
+    "prettier": "2.5.1",
+    "ts-jest": "27.1.0",
+    "typescript": "4.5.2"
+  },
+  "pnpm": {
+    "overrides": {
+      "react-scripts>eslint-plugin-testing-library": ">=5",
+      "react-scripts>@typescript-eslint/parser": ">=5",
+      "react-scripts>@typescript-eslint/eslint-plugin": ">=5"
+    }
```

Comment:
?

Reply:
Yeah, just conflicting versions between CRA and Galex. I've added a paragraph about this in the contributing guide.